runtime/missingmember_test.go

```diff
@@ -1697,7 +1697,7 @@ pub contract ItemNFT: NonFungibleToken {
    pub resource interface ItemCollectionPublic {
        pub fun deposit(token: @NonFungibleToken.NFT)
        pub fun batchDeposit(tokens: @NonFungibleToken.Collection)
-       pub fun getIDs(): [UInt64]
+       view pub fun getIDs(): [UInt64]
```

Comment:
Can the `getIDs` functions stay non-view, now that the function isn't used in a condition anymore?